pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -1000,12 +1000,13 @@ func AllReceivers(route *config.Route) (res []string) {
 }
 
 type GettableGrafanaReceiver struct {
-	UID                   string           `json:"uid"`
-	Name                  string           `json:"name"`
-	Type                  string           `json:"type"`
-	DisableResolveMessage bool             `json:"disableResolveMessage"`
-	Settings              *simplejson.Json `json:"settings"`
-	SecureFields          map[string]bool  `json:"secureFields"`
+	UID                   string            `json:"uid"`
+	Name                  string            `json:"name"`
+	Type                  string            `json:"type"`
+	DisableResolveMessage bool              `json:"disableResolveMessage"`
+	Settings              *simplejson.Json  `json:"settings"`
+	SecureFields          map[string]bool   `json:"secureFields"`
+	Provenance            models.Provenance `json:"provenance"`
```

Comment:
So for notif. policies, we tagged this field like `json:"provenance,omitempty"`, so the field won't exist at all if it's empty string. That way, unprovisioned objects are exactly the same as before, and we don't have fields appearing even with the feature flag off.

The side effect, is that we don't have to update all the expected responses in the tests with the new field. So, this could also reduce some changes here.

Reply:
Good catch, will change it 👍